bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -335,6 +362,9 @@ def _configure_vision(self, config, level):
                     # Set both ball and goalpost detector
                     self._ball_detector = yolo_handler.YoloBallDetector(config, self._yolo)
                     self._goalpost_detector = yolo_handler.YoloGoalpostDetector(config, self._yolo)
+                    # Check if we use the yolo robot detection
```

Comment:
What is the else case here? Use a previously instantiated robot detector?
Then we could mention that in the comment

Reply:
The whole part has been updated, but the idea is to create a model-based obstacle detector by default and substitute it with a YOLO based one if the model supports it.